packages/react-query/src/createTRPCReact.tsx

```diff
@@ -97,7 +97,7 @@ export type DecorateProcedure<
            */
           useInfiniteQuery: (
             input: Omit<inferProcedureInput<TProcedure>, 'cursor'>,
-            opts?: UseTRPCInfiniteQueryOptions<
+            opts: UseTRPCInfiniteQueryOptions<
```

Comment:
revisit

we briefly discussed wanting to abstract away the required `defaultPageParam` option or similar.